src/sentry/integrations/slack/utils/channel.py

```diff
@@ -65,7 +65,8 @@ def get_channel_id(
     # This means some users are unable to create/update alert rules. To avoid this, we attempt
     # to find the channel id asynchronously if it takes longer than a certain amount of time,
     # which I have set as the SLACK_DEFAULT_TIMEOUT - arbitrarily - to 10 seconds.
-    return get_channel_id_with_timeout(integration, channel_name, timeout)
+    ret = get_channel_id_with_timeout(integration, channel_name, timeout)
```

Comment:
@chloeho7 We should default to the old strategy if the feature flag `slack-use-new-lookup` is false. Example in code: https://github.com/getsentry/sentry/blob/master/src/sentry/api/bases/organization_events.py#L126-L128

Reply:
Do we need the actor kwarg for this case?

Reply:
@chloeho7 good question, no we do not. It can be omitted, the only impact is that you can't turn on a feature flag just for your user. It has to be at the organizational level.